config/remotePlugin.js

```diff
@@ -0,0 +1,54 @@
+const { resolve } = require('path');
+const packageInfo = require('../package.json');
+
+module.exports = {
+  pluginMetadata: {
+    name: packageInfo.name,
+    version: packageInfo.version,
+    exposedModules: {
+      init: resolve(__dirname, '../src/PluginEntry/PluginEntry.tsx'),
+    },
+  },
+  extensions: [
+    {
+      type: 'console.page/route',
```

Comment:
Do we still need `console.` prefixes? Has HAC-Core not worked away from this?

Reply:
Ah -- I guess I don't really know the goals of HAC-Infra. I assumed it was just a HAC Plugin, but I suppose if they wanted to share between the two apps, this would be needed. @rhamilto @jhadvig Any input here?

Reply:
I'll have to defer to @jhadvig here as I'm not sure what the plans are...

Reply:
Long term, we might do this, but by then hopefully we'll have updated OpenShfit console to use the HAC core SDK. More likely in the short term we'll be moving the other way and bringing OpenShift workload pages into HAC. I would remove the `console.` prefixed extensions for now. We can always add them later if needed.

Reply:
@spadgett @rhamilto I tried getting rid of the `console.` prefix for navigation but it did not work since HAC Core is currently implementing the `console.navigation` extension type and not `core.navigation` https://github.com/openshift/hac-core/blob/main/frontend/src/Navigation.ts#L26. 
We need to update HAC Core to handle the `core.navigation` extension. I'll run the change by the HAC Core team. Perhaps this comment can be addressed in a subsequent PR once HAC Core is updated.

Reply:
@spadgett I had a chat with @florkbr and he clarified that HAC Core still handles `console.navigation` extension for backward compatibility reasons for OpenShift Console based plugins. We will need to discuss the plan forward with Karel once he's back so for the time being there will be this duplication. But I'll put up a new PR when we have more clarity on this.